workers/unity/Packages/com.improbable.gdk.core/Config/RuntimeConfig.cs

```diff
@@ -18,12 +18,14 @@ public static class RuntimeConfigDefaults
     /// </summary>
     public static class RuntimeConfigNames
     {
+        public const string DeploymentTag = "deploymentTag";
```

Comment:
What is this?

Reply:
This is a config field specifically for the steam auth flow: https://docs.improbable.io/reference/13.3/csharpsdk/api-reference#improbable-worker-steamcredentials-class

Reply:
Would it be better for this be "steamDeploymentTag"?

Reply:
not really. A deployment tag is simply, as the name, a tag that can be added to a deployment. can be used to tag a deployment as a "steam deployment", but can also be used to tag other things.

Reply:
Hmm... it's confusing that it's being placed only within "steam.deploymenttag" but it looks like it's something that should be resolved in the worker side and not GDK

Reply:
I think there are good argument for naming this either way. Given that the deployment tag field is currently only ever used for the steam flow, I'll rename it to "steamDeploymentTag" to make the intent more clear.